core/bot.py

```
@@ -6,12 +6,24 @@
 from slack_bolt.adapter.socket_mode.aiohttp import AsyncSocketModeHandler
 from slack_sdk.errors import SlackApiError
 from core.workflows import PRWorkflows
-from services.approval_system import ApprovalService
+from services.developer.approval_system import ApprovalService
 from models.schemas import PRReviewRequest, PRCreationRequest, PRCommentHandlingRequest
 from config.settings import settings
 import re
 from utils.opik_tracer import trace
 import asyncio
```

Ruff (0.12.2)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)